### ISSUE TITLE
Improve help/FAQ pages to reduce impact on inbox

### ISSUE THREAD

**Issue by sabpop** [openaustralia/planningalerts#1700]
**What is it?**
A help section on the Planning Alerts site similar to [Right to Know Help section](https://www.righttoknow.org.au/help/about). It includes the existing FAQ, help articles and a link to the [Contact us page ](https://github.com/openaustralia/planningalerts/issues/1684). #1684 

**What we heard?**
- People are writing in to get help with a number of known issues, most of which are already written about in the FAQ. This means people are either not noticing it or are finding it difficult to navigate. 
- Many users are not aware of who Planning Alerts is and that it's not associated with government. 

**Read more in the User Experience Map on Miro here:**
https://miro.com/app/board/uXjVPrTUbvg=/ CLOSED
UPDATE View thinking here: https://www.figma.com/board/6Rc33vkc58nKooUkgE5w0i/Strategy%3A-Help-Pages?node-id=0-1

Focus on cards: Understanding the Planning Alerts  process, Hearing back from council, There's a problem, Checking coverage, Contacting PA thinking it's council, Clarity on who Planning Alerts is, Understanding the system. 

**Hypothesis**
This would:
- Support people to self serve
- Reduce amount of emails coming in about known issues
- Support people to understand the planning system and where Planning Alerts fits into it

**Design specifications**
- Draw attention to the difference between legislations in different states. We have examples in Right to know on this. eg. governed by which legislation a short description etc.
- Support with the next steps to contacting  planning authority eg. who to contact, how to contact them. Provide direct links to all planning authorities covered by planning alerts
- Make it clear that PA doesn't review the comments, that they go directly to the authority.
- Write a short explanation of why the complexity is hard to describe eg it makes sense you dont get it
- Include a list of helpful articles

**Implementation notes:**
Its a big piece of work it will involve a lot of work for us to research, write, iterate.
For iteration 1:
- How do we structure the help section:
1. What's an article.
2. How big is a help topic.
3. What topics
- Focus on answering the problems that people mail us for now
- Structure the UX in a way that is more approachable
4. Include previous work from https://github.com/openaustralia/planningalerts/issues/1595

**Sketch/ wireframe**
<img width="874" alt="Screenshot 2023-04-18 at 11 38 11 am" src="https://user-images.githubusercontent.com/127382981/232647525-73a02c2a-3390-4010-883a-b774cd7617b6.png">

**Comment by sabpop** [comment]
Before any other work, the first item to resolve here:
**How do we structure the help section:**
1. What's an article.
2. How big is a help topic.
3. What topics

**We can keep a parking lot of potential help articles in this thread:**
1. Guidelines for commenting

**Comment by katska** [comment]
@sabpop you mentioned in your round up in slack looking ahead to future work mapping help content. I'm so so so very keen to keep that trucking along. Could it work for you if we keep this issue to be the main thread as we navigate this process?

**Comment by sabpop** [comment]
@katska that works for me. also had a look at #1595 thanks for sharing that! that will be so helpful!!!!!!

**Comment by katska** [comment]
see also https://github.com/openaustralia/planningalerts/issues/1684

**Comment by JoannaHill** [comment]
This still needs attention and we've done lots of research.
